packages/tatum-neo/src/wallet/wallet.ts

```diff
@@ -0,0 +1,9 @@
+import Neon, { wallet } from '@cityofzion/neon-js'
+
+/**
+ * Generate Neo address and private key.
+ */
+export const generateNeoWallet = () => {
```

Comment:
```suggestion
export const generateNeoWallet = (): Promise<WalletWithAddress> => {
```
+ add to imports

Reply:
Type added, but without Promise as function is not asynchronous.